src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -1073,9 +1074,21 @@ def get_extension_class(extension_name: str):
     """
     global _possible_extensions
     extensions_dict = {ext.extension_name: ext for ext in _possible_extensions}
-    assert (
-        extension_name in extensions_dict
-    ), f"Extension '{extension_name}' is not registered, please import related module before use"
+
+    if extension_name not in extensions_dict:
+        if extension_name in _builtin_extensions:
+            module = _builtin_extensions[extension_name]
+            if autoload:
+                print(f"module '{module}' is imported automatically for extension '{extension_name}'")
+                imported_module = importlib.import_module(module)
+                extensions_dict = {ext.extension_name: ext for ext in _possible_extensions}
+            else:
+                raise ValueError(
+                    f"Extension '{extension_name}' is not registered, please import related module before use: 'import {module}'"
+                )
+        else:
+            raise ValueError(f"Extension '{extension_name}' is unkown maybe this is an external extension or a typo.")
```

Comment:
```suggestion
            raise ValueError(f"Extension '{extension_name}' is unknown maybe this is an external extension or a typo.")
```

This I really like since I have a tendency to type too fast in my terminal. Haha.